Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/LocalPerformanceContext.swift

```diff
@@ -0,0 +1,71 @@
+// swiftlint:disable force_try
```

Comment:
[aside] I know we do this everywhere in our tests, so I think it's fine for now but I thinkgoing forward we should try to avoid this. Right now, a lot of our tests fail with no explanation because of force-try failures, and more carefully catching errors in test setup would help us resolve strange issues.

Reply:
I agree, fixed this